tests/integration/passthrough-test.js

```diff
@@ -202,3 +202,25 @@ test('it can passthrough other-origin hosts', function(assert) {
   });
 });
 
+
+test('passthrough is not overridable', function(assert) {
+  assert.expect(1);
+  var done1 = assert.async();
+  var server = this.server;
+
+  server.loadConfig(function() {
+    this.urlPrefix = '/vet/123';
```

Comment:
why do you specify the `urlPrefix` here? As it is written this test will actually also pass on `master`, because of this. I think we can remove it.